src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -207,7 +209,8 @@ public String description() {
      * executed. The filter {@code operation} is not expected to handle the
      * exchange or {@linkplain HttpExchange#sendResponseHeaders(int, long) send the response headers}.
      * Doing so is likely to fail, since the exchange has commonly been handled
-     * before the operation is invoked.
+     * before the {@code operation} is invoked. More specifically, the response
+     * will typically be sent before the filter {@code operation} is executed.
      *
```

Comment:
I think I'd prefer "may be" rather than "will typically be" here as well. Best not to make any claim about what is more likely to happen.